integreat_cms/cms/constants/roles.py

```diff
@@ -57,6 +63,11 @@
     "view_poi",
 ]
 
+#: The permissions of the limited_page_manager
+LIMITED_PAGE_MANAGER_PERMISSIONS = EVENT_MANAGER_PERMISSIONS + [
+    "view_page",
```

Comment:
I'm not entirely sure about this. If I'm interpreting the issue correctly this is only about pages. So I don't think the limited page manager should inherit the permissions of the event manager. But maybe I'm incorrect about this